### PR TITLE
Fix sys/static.sh after 2d160f250918609fa8f399a33f003cedaa85d888

### PR DESCRIPTION
**Pull request by trufae**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

**Comment by ret2libc** [comment]
See static CI please.

File: sys/static.sh [hunk 1]
```diff
@@ -78,9 +81,30 @@ cat .test.c
...
 	CC=gcc
 fi
+PKG_CONFIG_FLAGS=`
+PKG_CONFIG_PATH="${PWD}/r2-static/usr/lib/pkgconfig" \
+pkg-config \
+  --define-variable="libdir=${PWD}/r2-static/usr/lib" \
+  --define-variable="prefix=${PWD}/r2-static/usr" \
+  --static --cflags --libs r_core
+`
+echo "${PKG_CONFIG_FLAGS}"
```
Comment (ret2libc) [inline sys/static.sh:91]: can't you use `set -x`?

File: sys/static.sh [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 [ -z "${STATIC_BINS}" ] && STATIC_BINS=0
 
+if [ 1 = 1 ]; then
```
Comment (ret2libc) [inline sys/static.sh:5]: remember to remove this before merging.

File: sys/static.sh (hunk 1 above)
Comment (trufae) [reply sys/static.sh]: done

**Comment by XVilka** [comment]
We also can build static binaries with Meson I think:
- https://mesonbuild.com/Build-targets.html
- https://renenyffenegger.ch/notes/development/languages/C-C-plus-plus/GCC/create-libraries/Meson/index

**Comment by trufae** [comment]
its green, what else do i have to look at?

**Pull request merged**